src/components/Todo/Todo.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { todoType } from '../propTypes/todoType';
+import { User } from '../User/User';
+
+import './Todo.scss';
+
+export const Todo = ({ title, user, completed }) => (
+  <div className="todos__item">
+    <strong><User name={user.name} /></strong>
```

Comment:
```suggestion
    <User name={user.name} />
```
лучше внутри User добавь этот тэг